src/hotspot/cpu/aarch64/templateInterpreterGenerator_aarch64.cpp

```diff
@@ -969,7 +969,7 @@ address TemplateInterpreterGenerator::generate_CRC32_update_entry() {
 /**
  * Method entry for static native methods:
  *   int java.util.zip.CRC32.updateBytes(int crc, byte[] b, int off, int len)
- *   int java.util.zip.CRC32.updateByteBuffer(int crc, long buf, int off, int len)
+ *   int java.util.zip.CRC32.updateByteBuffer(int crc, jlong buf, int off, int len)
```

Comment:
Why not `int64_t` instead of `jlong`?   Similarly in a couple later places.

Reply:
This comment refers to Java code which more appropriately uses jlong.  Actually, the Java code uses long.  I'll revert this change.

Reply:
Yeah, this is in a comment referring to a Java method signature, so I agree `long` is correct here.